src/tests/Common/Coreclr.TestWrapper/MobileAppHandler.cs

```diff
@@ -79,10 +79,7 @@ private static int HandleMobileApp(string action, string platform, string catego
                         {
                             cmdStr += $" --app=net.dot.{category}";
                         }
-                    }
 
-                    if (platform != "android")
-                    {
```

Comment:
So the next line is adding `--target ios-simulator-64` to `cmdStr`, but line 72 above already has `--target=ios-simulator-64` (equal sign instead of space).

Do we need both?  Can we remove one?